pkgs/applications/virtualization/krunvm/default.nix

```diff
@@ -22,17 +27,35 @@ stdenv.mkDerivation rec {
     hash = "sha256-3WiXm90XiQHpCbhlkigg/ZATQeDdUKTstN7hwcsKm4o=";
   };
 
-  nativeBuildInputs = with rustPlatform;[
+  nativeBuildInputs = with rustPlatform; [
     cargoSetupHook
     rust.cargo
     rust.rustc
     asciidoctor
-  ];
+    makeWrapper
+  ] ++ lib.optionals stdenv.isDarwin [ sigtool ];
 
-  buildInputs = [ libkrun ];
+  buildInputs = [ libkrun ] ++ lib.optionals stdenv.isDarwin [
+    libiconv
+  ];
 
   makeFlags = [ "PREFIX=${placeholder "out"}" ];
 
+  postInstall = ''
+    mkdir -p $out/etc
+    install -D -m755 ${buildah-unwrapped.src}/docs/samples/registries.conf $out/etc/containers/registries.conf
+    install -D -m755 ${buildah-unwrapped.src}/tests/policy.json $out/etc/containers/policy.json
+  '';
```

Comment:
These config files should reside in `/etc/containers`, not hard coded in packages.

Reply:
This is due to the logic of `krunvm` to find configuration:

https://github.com/containers/krunvm/blob/ba9e585085cd02d0912d0aa254617e4299ade2b2/src/utils.rs#L29-L38

Maybe we can patch this code and move that configuration elsewhere.

Reply:
Maybe open an issue upstream to ask about it.

Reply:
I have moved the files to `share/krunvm`.